google-cloud-firestore/src/main/java/com/google/cloud/firestore/AggregateQuery.java

```diff
@@ -25,32 +25,27 @@
 import com.google.api.gax.rpc.StreamController;
 import com.google.cloud.Timestamp;
 import com.google.cloud.firestore.v1.FirestoreSettings;
-import com.google.firestore.v1.RunAggregationQueryRequest;
-import com.google.firestore.v1.RunAggregationQueryResponse;
-import com.google.firestore.v1.RunQueryRequest;
-import com.google.firestore.v1.StructuredAggregationQuery;
-import com.google.firestore.v1.Value;
+import com.google.firestore.v1.*;
```

Comment:
Using `*` wildcard goes against style guide.

go/java-style#s3.3.1-wildcard-imports

Reply:
Thanks for catching this @tom-andersen . I believe my IDE has done this behind my back! 🙅
Fixed.